### PR TITLE
Send the auth header when deleting the users access token in helios

### PR DESCRIPTION
**Pull request by drsnyder**
Send the auth header when deleting the users access token in helios.

https://wikia-inc.atlassian.net/browse/SERVICES-707

/cc @Wikia/services-team

### CODE REVIEW

File: lib/Wikia/src/Service/Helios/HeliosClientImpl.php [hunk 1]
```diff
@@ -13,6 +13,8 @@ class HeliosClientImpl implements HeliosClient
 	const CLIENT_ID = "client_id";
 	const CLIENT_SECRET = "client_secret";
 
+	const HELIOS_AUTH_HEADER = "X-Wikia-UserId";
```
Comment (nmonterroso) [inline lib/Wikia/src/Service/Helios/HeliosClientImpl.php]: Can we change usage of https://github.com/Wikia/app/blob/dev/lib/Wikia/src/Service/Swagger/ApiProvider.php#L10 to use this instead?

**Comment by nmonterroso** [comment]
minor comment otw lgtm

**Comment by ArturKlajnerok** [comment]
Lgtm :+1:

File: lib/Wikia/src/Service/Helios/HeliosClientImpl.php (hunk 1 above)
Comment (drsnyder) [inline lib/Wikia/src/Service/Helios/HeliosClientImpl.php]: I saw that reference... Just so I'm clear, that's not auto generated?

Comment (drsnyder) [inline lib/Wikia/src/Service/Helios/HeliosClientImpl.php]: Doesn't look like it... I think I'm going to move this into `Wikia\Service\Constants`.

Comment (harnash) [inline lib/Wikia/src/Service/Helios/HeliosClientImpl.php]: @drsnyder, @nmonterroso: Added `Wikia\Service\Constants` and moved the `X-Wkia-UserId` there (a8cd822).

**Comment by harnash** [comment]
Implemented all the changes requested in comments.

@Wikia/services-team

**Comment by sqreek** [comment]
Lgtm

**Comment by nmonterroso** [comment]
@harnash the constant was moved out of `ApiProvider`, but looks like there are references to the constant in the `UserAvatarsApi`

File: lib/Wikia/src/Service/Helios/HeliosClientImpl.php [hunk 2]
```diff
@@ -155,16 +157,18 @@ public function refreshToken( $token )
 	 * A shortcut method for token invalidation requests.
 	 *
 	 * @param $token string - a token to be invalidated
+	 * @param $userId integer - the current user id
 	 *
 	 * @return string - json encoded response
 	 */
-	public function invalidateToken( $token )
+	public function invalidateToken( $token, $userId )
```
Comment (nmonterroso) [inline lib/Wikia/src/Service/Helios/HeliosClientImpl.php]: This is incompatible with the `HeliosClient` interface

**Pull request merged**